src/astro/databases/postgres.py

```diff
@@ -70,20 +73,31 @@ def load_pandas_dataframe_to_table(
         :param if_exists: Strategy to be used in case the target table already exists.
         :param chunk_size: Specify the number of rows in each batch to be written at a time.
         """
-        schema = None
         if target_table.metadata and target_table.metadata.schema:
             self.create_schema_if_needed(target_table.metadata.schema)
-            schema = target_table.metadata.schema.lower()
-        source_dataframe.to_sql(
-            target_table.name,
-            schema=schema,  # type: ignore
-            con=self.sqlalchemy_engine,
+
+        # create an empty table
+        db = SQLDatabase(engine=self.sqlalchemy_engine)
+        db.prep_table(
+            source_dataframe,
+            target_table.name.lower(),
+            schema=target_table.metadata.schema,
             if_exists=if_exists,
-            chunksize=chunk_size,
-            method="multi",
             index=False,
         )
 
+        output_buffer = io.StringIO()
+        source_dataframe.to_csv(output_buffer, sep=",", header=True, index=False)
+        output_buffer.seek(0)
+        table_name = self.get_table_qualified_name(target_table)
+        postgres_conn = self.hook.get_conn()
+        with closing(postgres_conn) as conn, closing(conn.cursor()) as cur:
+            cur.copy_expert(
+                f"COPY {table_name} FROM STDIN DELIMITER ',' CSV HEADER;",
```

Comment:
We'll probably want to allow users to customize the CSV delimiter and if the CSV has a header or not. That could be part of the `File` object.